src/tests/Home.test.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { render, getByTestId } from "@testing-library/react";
+import { Home } from "../components/Home";
+
+test("<Home/> component heading should be rendered in the page", () => {
+  const dispatch = jest.fn();
+  const { getByTestId } = render(<Home dispatch={dispatch} />);
+  const header = getByTestId("page-header");
+  expect(header).toBeInTheDocument();
+});
+
+test("<Home/> component should have 5 cards, information, schedule, price, theme and status", () => {
```

Comment:
are we testing the actual content that gets fetched and put into the store?

Reply:
At the moment just testing if there are 5 cards on the screen, not sure about testing with redux, action creator, something I will need to get some help and advice on integration testing with redux.

Reply:
These might help you: 
https://www.npmjs.com/package/redux-mock-store
https://www.npmjs.com/package/axios-mock-adapter